### PR TITLE
[NUI] Add Title in Wearable component

### PR DESCRIPTION
**Pull request by JoogabYun**
### Description of Change ###

 The Title inherits TextLabel.
 The Title can has a fadeOut effects on the left and right side.

### API Changes ###

 - ACR: NONE

### CODE REVIEW

File: src/Tizen.NUI.Wearable/src/public/Title.cs [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+            {
+                leftImage = new ImageView();
+                leftImage.PositionUsesPivotPoint = true;
+                leftImage.ParentOrigin = Tizen.NUI.ParentOrigin.TopLeft;
+                leftImage.PivotPoint = Tizen.NUI.PivotPoint.TopLeft;
+                leftImage.WidthResizePolicy = ResizePolicyType.FillToParent;
+                leftImage.HeightResizePolicy = ResizePolicyType.FillToParent;
+                this.Add(leftImage);
+            }
+            if (null == rightImage)
```
Comment (neostom432) [inline src/Tizen.NUI.Wearable/src/public/Title.cs:219]: Could you explain any case that these inner items are existing before initializing?

File: src/Tizen.NUI.Wearable/src/public/Title.cs [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+                {
+                    rightImage.Unparent();
+                    rightImage.Dispose();
+                    rightImage = null;
+                }
+            }
+            base.Dispose(type);
+        }
+
+        private void UpdateImage()
```
Comment (neostom432) [inline src/Tizen.NUI.Wearable/src/public/Title.cs:166]: When user change size of Title Control bigger, is blur updated?

File: src/Tizen.NUI.Wearable/src/public/Title.cs (hunk 1 above)
Comment (JoogabYun) [reply src/Tizen.NUI.Wearable/src/public/Title.cs:219]: There is not. This is an unnecessary check code!

File: src/Tizen.NUI.Wearable/src/public/Title.cs (hunk 2 above)
Comment (JoogabYun) [reply src/Tizen.NUI.Wearable/src/public/Title.cs:166]: Yes, The width is fixed and the height follows the Title Control height.

File: src/Tizen.NUI.Wearable/src/public/Title.cs [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+            }
+
+            this.Ellipsis = false;
+            this.HorizontalAlignment = HorizontalAlignment.Center;
+            this.VerticalAlignment = VerticalAlignment.Center;
+        }
+
+        private void Initialize()
+        {
+            leftImage = new ImageView();
```
Comment (neostom432) [inline src/Tizen.NUI.Wearable/src/public/Title.cs:209]: Minor comment
- we usually add m in front of member variable.

**Pull request merged**